TempleEight/docs/guide/orchestration.md

```diff
@@ -3,3 +3,174 @@ id: orchestration
 title: Orchestration
 sidebar_label: Orchestration
 ---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Considering how your application components are orchestrated is important as the components themselves.
+Temple provides several industry standards methods of automatically deploying your services.
+
+## Adding Orchestration to your Templefile
+l
+Orchestrating your project automatically requires only a single line of code in your Templefile.
+Working from the example in the [Getting Started](../getting-started) guide:
+
+```templefile {4}
+ExampleProject: project {
+  #language(go);
+  #database(postgres);
+  #provider(dockerCompose);
+}
+
+ExampleService: service {
+  foo: string;
+  bar: int;
+}
+```
+We can see here that the Templefile features a `#provider` annotation in the project block.
+This is how we specify that our project uses orchestration, the name in the argument specifies which framework we are generating for.
+
+The currently supported frameworks are:
+* Docker Compose
+* Kubernetes
```

Comment:
I think this needs to go in the introduction, and then you explain how to add it to the project